styles/core.scss

```diff
@@ -97,3 +97,33 @@ SPDX-License-Identifier: Apache-2.0 */
   flex-direction: column;
   flex: 1;
 }
+
+.side-interface {
+  height: 100vh;
+  display: flex;
+  flex-flow: column nowrap;
+  position:sticky;
+  top: 0px;
+  z-index:7;
+  flex:0;
+  overflow: hidden;
+  &.min {
+    min-width: v.$side-menu-minimised-width;
+    max-width: v.$side-menu-minimised-width;
+  }
+  &.max {
+    min-width: v.$side-menu-maximised-width;
+    max-width: v.$side-menu-maximised-width;
+  }
+  transition: all 0.5s cubic-bezier(0.1, 1, 0.2, 1);
+  @media (max-width: v.$page-width-medium-threshold){
+    position: fixed;
+    top:0;
+    &.zero {
```

Comment:
This does not need to be here. It can be `left: 0` by default, and the below class can simply overwrite the property if it is included in the classes.